WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/LoginFlow.java

```diff
@@ -1,32 +1,36 @@
 package org.wordpress.android.e2e.flows;
 
+import android.support.test.espresso.ViewInteraction;
+
 import org.wordpress.android.R;
-import org.wordpress.android.e2e.pages.MePage;
 
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
 import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_EMAIL;
 import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_PASSWORD;
-import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_USERNAME;
 import static org.wordpress.android.support.WPSupportUtils.clickOn;
+import static org.wordpress.android.support.WPSupportUtils.getCurrentActivity;
 import static org.wordpress.android.support.WPSupportUtils.isElementDisplayed;
 import static org.wordpress.android.support.WPSupportUtils.populateTextField;
+import static org.wordpress.android.support.WPSupportUtils.sleep;
```

Comment:
There's another comment below, but let's avoid the use of `sleep` directly if we can? In some cases it's necessary for UI reasons (in which case you could use `waitOneFrame`), but for network requests it could cause flakiness.

Reply:
I'll rework it without the sleep, I was just being accommodating of them for now since network mocking should drastically reduce the need for them, I tried `waitOneFrame` and it didn't work for me but I'll rework it